daemons/fenced/fenced_remote.c

```diff
@@ -582,6 +627,9 @@ finalize_op(remote_fencing_op_t *op, xmlNode *data, bool dup)
         /* Defer notification until the bcast message arrives */
         fenced_broadcast_op_result(op, op_merged);
         free_xml(local_data);
+
+        /* Monitor response Clear data. */
+        clear_check_async_reply(op);
```

Comment:
It occurs to me that if we do clear_check_async_reply() in clear_remote_op_timers(), we don't have to do it anywhere else, and it helps us in case the op gets freed somewhere unexpected.

Reply:
In check_async_reply_cb(), the parameter that calls finalize_op() is the data of check_async_reply_t.
When clear_check_async_reply() is executed with clear_remote_op_timers(), the data of check_async_reply_t will be cleared and the subsequent processing of finalize_op() will fail.
Therefore, the timing to execute clear_check_async_reply() must be just before exiting finalize_op().

Reply:
But....
I may be able to change to such processing by changing the way I hold the data now, so I will check it.

Reply:
Since the message that failed fencing is saved only in msg of check_async_reply_t, it cannot be obtained from stonith_remote_op_list, so I think that the current process is correct.